src/jit/compiler.cpp

```diff
@@ -4442,13 +4442,6 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, CORJIT_F
 #endif
 
 #ifndef LEGACY_BACKEND
-    // TODO-CQ: Remove "if-block" when lowering doesn't rely on front end liveness.
-    // Remove "if-block" to repro assert('!"We should never hit any assignment operator in lowering"')
-    // using self_host_tests_amd64\JIT\Methodical\eh\finallyexec\tryCatchFinallyThrow_nonlocalexit1_d.exe
-    if (!fgLocalVarLivenessDone)
-    {
-        fgLocalVarLiveness();
-    }
```

Comment:
I don't have any recollection of the Rationalizer or Lower using DF. They don't seem to have any references to the bbLive sets.

---
In reply to: [76298448](https://github.com/dotnet/coreclr/pull/6552#discussion_r76298448) [](ancestors = 76298448)